Share View/mashup.config.js

```diff
@@ -1,5 +1,4 @@
 tau.mashups.addModule("ShareBoard/config", {
     "serviceUrl": "https://tauboard.com",
-    "title": "Share %s",
```

Comment:
Was `%s` replaced by exact view type - list, view, timeline, etc? So now we can share only views?

Reply:
"view" is a general concept which includes boards, lists, one-by-one, timelines, reports and dashboards